napalm/base.py

```diff
@@ -364,3 +364,44 @@ def get_interfaces_counters(self):
             }
         """
         raise NotImplementedError
+
+        def show_route(self, destination = ''):
```

Comment:
Ok! In that case we could create an exception called IPv6NotImplemented and use it for cases like this. I still would like to change the name though, but to `get_routes_to` if we are going to support both v4 and v6. The reason is that the rest of the methods are named get_something and I would like to keep it consistent.